src/orderbook.js

```diff
@@ -828,6 +828,32 @@ OrderBook.prototype.isBalanceChangeNode = function(node) {
   return true;
 };
 
+OrderBook.prototype._canRunAutobridgeCalc = function(): boolean {
+  return !this._calculatorRunning && (Date.now() - this._lastCalcDuration >= 100);
```

Comment:
The browser should have a change to repaint and handle user interaction every time you call the `setTimeout(fn, 0)`, so I don't think we need to worry about it. Also, I think the logic is incorrect because it is comparing an absolute time with a duration.